### PR TITLE
Fix the month value to display. (for codelab-complete)

### PR DESCRIPTION
**Pull request by yoichiro**
Same as #18 for `codelab-complete` branch.

### CODE REVIEW

File: app/src/main/java/com/devrel/android/fitactions/home/FitStatsAdapter.kt [hunk 1]
```diff
@@ -75,7 +75,7 @@ class FitStatsAdapter : ListAdapter<FitActivity, FitStatsAdapter.ViewHolder>(DIF
                 R.string.stat_date,
                 day,
                 calendar.get(Calendar.DAY_OF_MONTH),
-                calendar.get(Calendar.MONTH)
+                calendar.get(Calendar.MONTH) + 1
```
Comment (dzuluaga) [inline app/src/main/java/com/devrel/android/fitactions/home/FitStatsAdapter.kt:78]: Good catch. What are your thoughts of using simpledateformat instead? Thanks!

var month = SimpleDateFormat("MM")

month.format(calendar.time).toInt()

Comment (yoichiro) [reply app/src/main/java/com/devrel/android/fitactions/home/FitStatsAdapter.kt:78]: Yes, I agree your suggestion. We can give expression the intent to display the month value using the `SimpleDateFormat` more clearly than using `+1`. I intend to modify the code soon. Thank you for your advice! :)

Comment (yoichiro) [reply app/src/main/java/com/devrel/android/fitactions/home/FitStatsAdapter.kt]: I changed the code based on your advice: 0eee99c Could you review the diff again?

**Comment by yoichiro** [comment]
@dzuluaga Thank you for the reviewing and merging!

**Comment by dzuluaga** [comment]
No, thank you Yoichiro!

**Pull request merged**